examples/csi-config-map-sample.yaml

```diff
@@ -13,8 +13,8 @@ kind: ConfigMap
 # To add more clusters or edit MON addresses in an existing configmap, use
 # the `kubectl replace` command.
 # The <rados-namespace> is optional and represents a radosNamespace in the pool.
-# If any given all the of the rbd images, snapshots and other metadata
-# will be store within the radosNamespace.
+# If any given, all of the rbd images, snapshots, and other metadata will be stored
+# within the radosNamespace.
```

Comment:
This now fails `yamlllint` with the following error:
```
 ./examples/csi-config-map-sample.yaml
  16:81     error    line too long (83 > 80 characters)  (line-length)
```

You can run `make containerized-test` to run the verification locally.